### PR TITLE
Fixes X-axis issue and adds formatting to plot axes using the scales …

### PR DESCRIPTION
**Pull request by JeffreyLambert**
…package

### CODE REVIEW

**Comment by JeffreyLambert** [comment]
Hopefully this gets you started.  The issue seemed to be that the game_num was a categorical variable and not a numeric variable.  Casting it back to a numeric seemed to fix the issue.

File: nba_career_pts/NBA_Projections.R [hunk 1]
```diff
@@ -6,6 +6,7 @@ library(rvest)
 library(dplyr)
 library(ggplot2)
 library(data.table)
+library(scales)
```
Comment (JeffreyLambert) [inline nba_career_pts/NBA_Projections.R:9]: Needs this package for pretty formatting of axes

File: nba_career_pts/NBA_Projections.R [hunk 2]
```diff
@@ -89,12 +90,19 @@ career_pts$g_date = as.Date(career_pts$date)
...
+
 # Create plot
-ggplot(data=career_pts, aes(x=factor(game_num), y=csum, color=playername)) +
-  geom_point(size=1, color='red') +
-  # scale_x_continuous(breaks=seq(0,1400,100)) +
-  theme_minimal() +
-  labs(x = "Game #", y = "Career Points")
-  # shadow_mark()
+career_pts %>%
+  mutate(game_num = as.numeric(game_num)) %>%
```
Comment (JeffreyLambert) [inline nba_career_pts/NBA_Projections.R:96]: This is the line which fixed up the X-axis issue.

**Review by gshick — APPROVED** [review]
Thanks for the assist (no pun intended)!

**Pull request merged**